configure.ac

```diff
@@ -126,8 +126,31 @@ AC_ARG_ENABLE(
 	AS_HELP_STRING([--enable-bsdtar], [Use alternative bsdtar command (uses tar by default)])
 )
 AS_IF([test "x$enable_bsdtar" = "xyes" ],
-	[AC_DEFINE(SWUPD_WITH_BSDTAR, 1, [Use bsdtar])],
-	[AC_DEFINE(SWUPD_WITHOUT_BSDTAR, 1, [Use default tar])]
+	[AC_DEFINE(SWUPD_WITH_BSDTAR, 1, [Use bsdtar])]
+)
+dnl Enable extended attribute support
+XATTR="yes"
+AC_ARG_ENABLE(
+	[xattr],
+	AS_HELP_STRING([--enable-xattr],[Use extended file attributes (unused by default)])
+)
+AS_IF([test "x$enable_xattr" = "xyes"],
+	    [AC_DEFINE(SWUPD_WITH_XATTRS,1,[Use extended file attributes])
+	    AS_IF(test "x$enable_bsdtar" = "xyes",
+	    echo "Options --enable-bsdtar and --enable-xattr are incompatible" >&2
+	    AS_EXIT(1))],
+	    [XATTR=no]
+)
+TARSELINUX="yes"
+AC_ARG_ENABLE([tar-selinux],
+	AS_HELP_STRING([--enable-tar-selinux],[give --selinux option to tar])
+)
+AS_IF([test "x$enable_tar_selinux" = "xyes"],
+	    [AC_DEFINE(SWUPD_TAR_SELINUX,1,[give --selinux option to tar])
+	    AS_IF(test "x$enable_xattr" = "xyes",,
```

Comment:
I think this line would be clearer if written as:

```
AS_IF(test "x$enable_xattr" != "xno",
```
so that the empty second argument to AS_IF is not needed.